### PR TITLE
Docs: Clarify Documentation Information for Undertale, Terraria, DOOM 1993

### PR DESCRIPTION
**Pull request by nicholassaylor**
### Undertale
- Standardized file names to match style of other games
- Removed redundant reference to Undertale in setup guide URL
- Emphasized existance of Undertale install in Archipelago folder
- Used codeblocks to better guide to user during installation
- Cleaned up sentences to be more concise/informative
- Other small formatting changes

### Terraria
- Outlined mod installation process in more detail
- Included information about downpatching to TModLoader for 1.4.3
- Other small edits

### DOOM 1993
- Small edits

## How was this tested?
Running Webhost and viewing updated guides on the website

### CODE REVIEW

File: worlds/terraria/docs/setup_en.md
Comment (kindasneaki) [inline worlds/terraria/docs/setup_en.md:22]:
```suggestion
4. TModLoader will say that it needs to refresh; exit this menu, and it will do this automatically
5. Once TModLoader finishes loading, the Archipelago mod is finished installing and you can now 
[connect to an Archipelago game](#joining-an-archipelago-game-in-terraria).
```

**Comment by kindasneaki** [comment]
@Daivuk  @Seldom-SE @jonloveslegos Since this involves your games

File: worlds/doom_1993/docs/setup_en.md
Comment (Daivuk) [inline worlds/doom_1993/docs/setup_en.md:1]: That looks fine and shouldn't conflict with my other PR 👍

**Review by Daivuk — APPROVED** [review]
For the DOOM part

File: worlds/terraria/docs/setup_en.md
Comment (Seldom-SE) [inline worlds/terraria/docs/setup_en.md:11]: tModLoader is spelled with a lowercase t

File: worlds/terraria/docs/setup_en.md
Comment (Seldom-SE) [inline worlds/terraria/docs/setup_en.md:12]: Unless this is standard Archipelago style, this should use quotes, since it's prose. Backticks are for code formatting. So, "Workshop" > "Manage Mods"

File: worlds/terraria/docs/setup_en.md
Comment (Seldom-SE) [inline worlds/terraria/docs/setup_en.md:16]: As before, "Betas" > "Beta Participation"

File: worlds/terraria/docs/setup_en.md
Comment (Seldom-SE) [inline worlds/terraria/docs/setup_en.md:31]: "May be used to break" is better, since it conveys that the progression-breaking is up to the player perform. The mod won't break progression unless the player uses it to do so.

File: worlds/terraria/docs/setup_en.md
Comment (Seldom-SE) [reply worlds/terraria/docs/setup_en.md:16]: And so on

**Review by Seldom-SE — CHANGES_REQUESTED** [review]

File: worlds/terraria/docs/setup_en.md
Comment (nicholassaylor) [reply worlds/terraria/docs/setup_en.md]: e9183cb7cbe6d0d2a64eae3cd82785d0543d90bf fixes this typo  throughout the guide.
tModLoader was capitalized in the guide when I found it, so I followed that. All previous mentions of "TModLoader" have been eliminated.

File: worlds/terraria/docs/setup_en.md
Comment (nicholassaylor) [reply worlds/terraria/docs/setup_en.md]: Change mostly reverted in e9183cb7cbe6d0d2a64eae3cd82785d0543d90bf  
Made change "May -> Can" to reinforce the idea

File: worlds/terraria/docs/setup_en.md
Comment (nicholassaylor) [reply worlds/terraria/docs/setup_en.md:12]: I will delegate this decision to @LegendaryLinux since he is the primary maintainer of the website.

File: worlds/terraria/docs/setup_en.md
Comment (nicholassaylor) [reply worlds/terraria/docs/setup_en.md:16]: See https://github.com/ArchipelagoMW/Archipelago/pull/2149#discussion_r1320794135

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [reply worlds/terraria/docs/setup_en.md:12]: I believe the convention is to use backticks (code) to indicate either code directly, or information that can be found within a technical file. As this is an option within a game menu, I think quotes are appropriate here.

**Comment by nicholassaylor** [comment]
@Seldom-SE addressed everything in your review, just asking for one last look over to make sure everything looks good to you.

**Review by Seldom-SE — APPROVED** [review]

**Comment by LegendaryLinux** [comment]
Has @jonloveslegos had a chance to review this?

File: worlds/doom_1993/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/doom_1993/docs/setup_en.md:14]: This change does not provide additional clarity. I recommend:
> Copy `DOOM.WAD` from your game's installation directory into the newly extracted folder.

File: worlds/doom_1993/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/doom_1993/docs/setup_en.md:16]: Code blocks should be reserved for referencing code, filenames, or other explicit system information. See my above usage of a code block to indicate an explicit filename. Menu items were correctly emphasized with bold previously.

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/terraria/docs/setup_en.md:12]: This should be emphasized with bold text:
3. Go to **Workshop -> Manage Mods** and enable the Archipelago mod.

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/terraria/docs/setup_en.md:16]: Use bold text instead of quotes to indicate a series of menu selections.

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/terraria/docs/setup_en.md:24]: This was a change in voice only. I discourage changes like this.

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/terraria/docs/setup_en.md:31]: While I think this was intended as another voice change, you have technically corrected grammar.
"May" indicates permission while "Can" indicates possibility.

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/terraria/docs/setup_en.md:40]: This is a voice change again, but both versions' grammar leaves something to be desired. A more correct version:

The [basic multiworld setup guide](/tutorial/Archipelago/setup/en) can be found on Archipelago's website. Among other things, it explains what `.yaml` files are, and how they are used.

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/terraria/docs/setup_en.md:50]: Menu navigation should be bold.

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/terraria/docs/setup_en.md:51]: Menu options should be bold.

File: worlds/terraria/docs/setup_en.md
Comment (LegendaryLinux) [inline worlds/terraria/docs/setup_en.md:52]: Menu options should be bold.

**Review by LegendaryLinux — CHANGES_REQUESTED** [review]

**Comment by LegendaryLinux** [comment]
I don't think were likely to get final confirmation from @jonloveslegos soon, so I will approve this after some style changes are made. See the review I posted.

File: worlds/terraria/docs/setup_en.md
Comment (Seldom-SE) [reply worlds/terraria/docs/setup_en.md:31]: In this particular case, it was intentional

**Review by LegendaryLinux — APPROVED** [review]

**Pull request merged**
